### PR TITLE
Fix issues seen since namespace rename

### PR DESCRIPTION
**Pull request by riarenas**
Fix file formatting from Namespace rename, fix issues caused by incorrect names in InspectableClass macros

### CODE REVIEW

File: source/uwp/Renderer/lib/WholeItemsPanel.h [hunk 1]
```diff
@@ -5,12 +5,11 @@
 
 namespace AdaptiveCards { namespace Rendering { namespace Uwp
 {
-
     class WholeItemsPanel : public Microsoft::WRL::RuntimeClass<ABI::AdaptiveCards::Rendering::Uwp::IWholeItemsPanel,
         ABI::Windows::UI::Xaml::IFrameworkElementOverrides,
         Microsoft::WRL::ComposableBase<ABI::Windows::UI::Xaml::Controls::IPanelFactory>>
     {
-        InspectableClass(L"AdaptiveCards.Uwp.WholeItemsPanel", BaseTrust)
+        InspectableClass(L"AdaptiveCards.Rendering.Uwp.WholeItemsPanel", BaseTrust)
```
Comment (riarenas) [inline source/uwp/Renderer/lib/WholeItemsPanel.h]: >AdaptiveCards.Rendering.Uwp.WholeItemsPanel [](start = 27, length = 43)

It seems this was the root cause of the issue. This resulted in the C# projection not being generated correctly, which didn't allow the test app to gather all the xaml image elements inside the panel. That resulted in not waiting for the images to render before creating the image for the test comparison.

Comment (riarenas) [reply source/uwp/Renderer/lib/WholeItemsPanel.h]: This absolutely didn't take me a whole day to figure out. Nope. Not at all.

---
In reply to: [154244385](https://github.com/Microsoft/AdaptiveCards/pull/1013#discussion_r154244385) [](ancestors = 154244385)

**Comment by RebeccaAnne** [comment]
> 

[nit] extra space (here and in several other of these files)

---
Refers to: source/uwp/Renderer/lib/AdaptiveCardParseResult.cpp:66 in 7961684. [](commit_id = 7961684cb91c312570691b4a40549964e1d59713, deletion_comment = False)

File: source/uwp/Renderer/lib/AdaptiveRenderArgs.h [hunk 2]
```diff
@@ -31,3 +30,4 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
 
     ActivatableClass(AdaptiveRenderArgs);
 }}}
+#pragma once
```
Comment (RebeccaAnne) [inline source/uwp/Renderer/lib/AdaptiveRenderArgs.h]: ?

Comment (riarenas) [reply source/uwp/Renderer/lib/AdaptiveRenderArgs.h]: I have no idea? Thanks for catching it

---
In reply to: [154391331](https://github.com/Microsoft/AdaptiveCards/pull/1013#discussion_r154391331) [](ancestors = 154391331)

**Comment by riarenas** [comment]
> 

Discussed offline. Seems like best practice is to end all your files with a newline character. Investigating ways to automate this.

---
In reply to: [348544104](https://github.com/Microsoft/AdaptiveCards/pull/1013#issuecomment-348544104) [](ancestors = 348544104)

---
Refers to: source/uwp/Renderer/lib/AdaptiveCardParseResult.cpp:66 in 7961684. [](commit_id = 7961684cb91c312570691b4a40549964e1d59713, deletion_comment = False)

**Pull request merged**